src/GeneralStateTestsFiller/stBadOpcode/badOpcodesFiller.json

```diff
@@ -37,7 +24,7 @@
 						},
 						{
                 "indexes" : {
-                    "data" : [115],
+                    "data" : [114],
```

Comment:
where is the expect section for all other data for network `>=Constantinople` ?
if there is no expect section, then there will be no test generated. 
(for example there will be no test for data 100 and network Constantinople)

Reply:
Right.